### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.4.3][IntelliJ][Wipro]: Keyboard focus order is illogical while navigating in the signin dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1434]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Navigate to Tools menu, point to "Azure", and then select "Azure Sign In".
4. In the Azure Sign In dialog navigate and observe the focus order.

**Expected Result:**
Keyboard focus order should be :
Interactive->Automated->Authentication edit box->Browse button->New button->Steps link->sign In ->Cancel button.

**Actual Result:**
 Keyboard focus order is illogical while navigating in the sign in dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![signin-no tab order](https://user-images.githubusercontent.com/22956130/35726249-151807bc-07b9-11e8-9037-0fe7b7be197f.jpg)

**Comment by Eskibear** [comment]
fixed in #1620 , will be shipped in next release.

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.7.0 .Issue got fixed.
Please add closed tag 
![1434-fixed](https://user-images.githubusercontent.com/22956130/39859104-519953e6-5456-11e8-99ed-4fe173f4403e.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749906
